scapy/sendrecv.py

```diff
@@ -359,8 +359,7 @@ def sr(x, promisc=None, filter=None, iface=None, nofilter=0, *args,**kargs):
 multi:    whether to accept multiple answers for the same stimulus
 filter:   provide a BPF filter
 iface:    listen answers only on the given interface"""
-    if "timeout" not in kargs:
-        kargs["timeout"] = -1
+    kargs.setdefault("timeout", -1)
```

Comment:
In fact, setting timeout to -1 is useless.
Those changes have been removed in https://github.com/secdev/scapy/pull/1142

To avoid conflicts, please do not edit it

Reply:
Done